snyk/snyk-ls
# fix: clipboard package not working with CGO_ENABLED=0 [ROAD-1185]

### Description

_This PR replaces the clipboard package due to compatibility issues when building with `CGO_ENABLED=0`. Before merging we will need to verify that the changes work on all platforms._

### Validation procedure

- Use goreleaser to build LS binaries in golang docker container:
```
docker run --rm -it -v ${PWD}:/app -w /app golang
go install github.com/goreleaser/goreleaser@latest
goreleaser build --rm-dist --snapshot
```
- Wire built binary as custom LS path in VSCode
- Open extension and try authenticating from scratch
- Press “Copy authentication link to clipboard”
- Auth link should be copied to clipboard (paste to verfiy)

### Checklist

- [x] Validated on darwin_arm64
- [ ] Validated on darwin_amd64
- [ ] Validated on linux_386
- [ ] Validated on linux_amd64
- [x] Validated on linux_arm64 (with `xsel` installed)
- [x] Validated on windows_386
- [ ] Validated on windows_amd64
- [x] License file updated, if new 3rd-party dependency is introduced
